### PR TITLE
move delayed_job logs to delayed_job.log

### PR DESCRIPTION
**Pull request by dwradcliffe**
This moves delayed_job logs to their own file so we can handle them differently.

@arthurnn @evanphx @qrush

### CODE REVIEW

**Comment by bradly** [comment]
@dwradcliffe Does log rotation need to be setup for this?

**Comment by dwradcliffe** [comment]
@bradly Yep, definitely does. We should be rotating `*.log` in that directory. We can still merge this PR first though.

**Comment by bradly** [comment]
@dwradcliffe Sounds good.

File: config/initializers/delayed_job.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require 'delayed_job'
 Delayed::Worker.max_attempts = 3
 Delayed::Worker.max_run_time = 5.minutes
+Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
```
Comment (qrush) [inline config/initializers/delayed_job.rb]: Could do `Rails.root.join("log/delayed_job.log")` ...if `Logger.new` can take a `Pathname`. If not then just `to_s` it.

**Comment by qrush** [comment]
:ok_hand:

**Comment by arthurnn** [comment]
LGTM

**Pull request merged**